websites/D/discord-botlist/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "MauricePascal",
+      "id": "622784776234991626"
+    },
+    "url": ["discord-botlist.eu", "www.discord-botlist.eu", "status.discord-botlist.eu"],
+    "description": {
+      "en": "Your Discord Bot List! We give you the ability to apply your bot and give your bot a page on our website."
+    },
+    "service": "discord-botlist",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/GqUUpuX",
+    "thumbnail": "https://i.imgur.com/GqUUpuX",
```

Comment:
Make the thumbnail a screenshot of the website,.

Reply:
![image](https://user-images.githubusercontent.com/20159369/89137861-2e7e4e80-d4ff-11ea-9fcc-c48883d54085.png)
also fix this ^^